src/router/routes.ts

```diff
@@ -3,36 +3,59 @@
 // import StateExample from '@/views/StateExample.vue'
 import DefineCompany from '@/views/DefineCompany.vue'
 import ReviewConfirm from '@/views/ReviewConfirm.vue'
+import SigninView from '@/views/auth/Signin.vue'
+import SignoutView from '@/views/auth/Signout.vue'
 
 export const routes = [
   {
     path: '/',
     redirect: '/define-company'
   },
-  // {
-  //   path: '/home',
-  //   name: 'home',
-  //   component: Home,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/MixinExample',
-  //   name: 'mixinExample',
-  //   component: MixinExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/StateExample',
-  //   name: 'stateExample',
-  //   component: StateExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
+  {
+    path: '/signin/:idpHint',
+    name: 'signin',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl',
+    name: 'signin-redirect',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl/:redirectUrlLoginFail',
+    name: 'signin-redirect-fail',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signout',
```

Comment:
1. How do we get to this signout route (page)?

2. Does this page require auth (see line 56)? ie, do we need to be logged in before we can go to the signout page?

Reply:
I think we probably don't need to specify that the signout page requires auth, but it doesn't break anything either way, so this is OK for now and we can change it later if needed.